tools/clang/test/CodeGenHLSL/mesh-val/multipleSetMeshOutputCounts.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %dxc -E main -T ms_6_5 %s | FileCheck %s
+// RUN: %dxilver 1.9 | %dxc -E main -T ms_6_5 %s | FileCheck %s
```

Comment:
Hang on, is CodeGenHLSL run via lit shell testing?  If so, this needs to be changed to `// REQUIRES: dxil-1-9` instead, right?

Reply:
There is a file in CodeGenHLSL that disables the directory from being discovered by lit, which implies this is not a lit shell testing directory.